src/Analysis/Codelyzer.Analysis.Build/ProjectBuildHandler.cs

```diff
@@ -175,6 +215,39 @@ public async Task<ProjectBuildResult> Build()
 
             return projectBuildResult;
         }
+
+        public async Task<ProjectBuildResult> SyntaxOnlyBuild()
+        {
+            await SetSyntaxCompilation();
```

Comment:
If `SetSyntaxCompilation` is changed from async to a synchronous method, we won't need `await` here.

Reply:
Ok